Doc/whatsnew/3.12.rst

```diff
@@ -869,6 +869,10 @@ Build Changes
   optimization levels (0, 1, 2) at once.
   (Contributed by Victor Stinner in :gh:`99289`.)
 
+* ``#pragma``-based linking with ``python3*.lib`` can now be switched off
+  with :c:expr:`Py_NO_LINK_LIB`. (Contributed by Jean-Christophe
+  Fillion-Robin in :gh:`82909`.)
```

Comment:
The main branch is now Python 3.14, so you should update Doc/whatsnew/3.14.rst instead, no?